models/spike_generator.h

```diff
@@ -252,8 +237,8 @@ class spike_generator : public DeviceNode
     //! Shift spike times at present to next step
     bool shift_now_spikes_;
 
-    Parameters_();                               //!< Sets default parameter values
-    Parameters_( const Parameters_& ) = default; //!< Recalibrate all times
+    Parameters_();                     //!< Sets default parameter values
+    Parameters_( const Parameters_& ); //!< Recalibrate all times
```

Comment:
The copy constructor should remain defaulted. In C++ , implicit declaration of `operator=` has been deprecated when a user-provided copy constructor is defined.

Reply:
Done